usersession/client/client.go

```diff
@@ -307,6 +312,59 @@ func (client *Client) ServicesStop(ctx context.Context, services []string) (stop
 	return stopFailures, err
 }
 
+func (client *Client) ServicesRestart(ctx context.Context, services []string) (restartFailures []ServiceFailure, err error) {
+	restartFailures, _, err = client.serviceControlCall(ctx, "restart", services)
+	return restartFailures, err
+}
+
+func (client *Client) ServicesReloadOrRestart(ctx context.Context, services []string) (restartFailures []ServiceFailure, err error) {
+	restartFailures, _, err = client.serviceControlCall(ctx, "reload-or-restart", services)
+	return restartFailures, err
+}
+
+// UserServiceUnitStatus corresponds to the systemd.UnitStatus in order to be able
+// to reliable transport the data we need surrounding the members we are interested in
```

Comment:
this comment is odd, what does surrounding mean here?  would something like

// ServiceUnitStatus is a JSON enconding capable representation of the systemd.UnitStatus for a user service.

Reply:
Thanks for the suggestion here, I agree my original comment was perhaps a bit weird. It sounded better in my head when I wrote it